subsys/bluetooth/mesh/Kconfig

```diff
@@ -32,6 +32,15 @@ config BT_MESH_PB_ADV
 	  Enable this option to allow the device to be provisioned over
 	  the advertising bearer.
 
+config BT_MESH_PROVISIONER
+	bool "Provisioner support"
+	select BT_ECC
+	select BT_MESH_PROV
+	select BT_MESH_NODES
```

Comment:
Do you really need the separate `BT_MESH_NODES` option? In what kind of scenario would one want to enable `BT_MESH_NODES` but not enable `BT_MESH_PROVISIONER`?

Reply:
It's not really needed no, it just felt like a natural split for some reason and also it is currently two separate commits. I can merge them and just leave the BT_MESH_PROVISIONER.